spring-security-modules/spring-security-authorization/spring-security-annotation-template-parameter/pom.xml

```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you remove identation of lines 3,4 below?

Reply:
i fixed it, thanks!